packages/serum/src/market.ts

```diff
@@ -1773,10 +1782,13 @@ export class OpenOrders {
     ownerAddress: PublicKey,
     newAccountAddress: PublicKey,
     programId: PublicKey,
+    seed: string,
   ) {
-    return SystemProgram.createAccount({
+    return SystemProgram.createAccountWithSeed({
       fromPubkey: ownerAddress,
+      basePubkey: ownerAddress,
```

Comment:
shouldn't this be `this._programid`?

Reply:
The docs says that => `basePubkey` Base public key to use to derive the address of the created account. Must be the same as the base key used to create newAccountPubkey

Reply:
yeh https://solana-labs.github.io/solana-web3.js/types/CreateAccountWithSeedParams.html
look down on line 1796 that is the program id

Reply:
+ i guess if he was passing the wrong stuff this call would fail because newAccountAddress wouldn't match what it was supposed to be and instruction would error

Reply:
yeah and also that whole thing works on the Solape DEX UI without changing anything (except of course `yarn link`ing the module)